pbs/pbsrequest.go

```diff
@@ -30,20 +37,42 @@ type Bids struct {
 	Params     json.RawMessage `json:"params"`
 }
 
+type PBSVideo struct {
+	Mimes          []string `json:"mimes,omitempty"`
+	Minduration    uint64   `json:"minduration,omitempty"`
+	Maxduration    uint64   `json:"maxduration,omitempty"`
+	Startdelay     int64    `json:"startdelay,omitempty"`
+	Skippable      int      `json:"skippable,omitempty"`
+	PlaybackMethod int8     `json:"playback_method,omitempty"`
+	Frameworks     []string `json:"frameworks,omitempty"`
+}
+
 type AdUnit struct {
-	Code     string           `json:"code"`
-	TopFrame int8             `json:"is_top_frame"`
-	Sizes    []openrtb.Format `json:"sizes"`
-	Bids     []Bids           `json:"bids"`
-	ConfigID string           `json:"config_id"`
+	Code       string           `json:"code"`
+	TopFrame   int8             `json:"is_top_frame"`
+	Sizes      []openrtb.Format `json:"sizes"`
+	Bids       []Bids           `json:"bids"`
+	ConfigID   string           `json:"config_id"`
+	MediaTypes []string         `json:"media_types"`
 }
 
 type PBSAdUnit struct {
-	Sizes    []openrtb.Format
-	TopFrame int8
-	Code     string
-	BidID    string
-	Params   json.RawMessage
+	Sizes      []openrtb.Format
+	TopFrame   int8
+	Code       string
+	BidID      string
+	Params     json.RawMessage
+	Video      PBSVideo
+	MediaTypes []MediaType
```

Comment:
Not sure of myself here... but does this make sense as a data model? It seems to me like `Video` should always be defined if `MediaTypes` includes `MEDIA_TYPE_VIDEO`, and always be `nil` if `MediaTypes` _doesn't_ include `MEDIA_TYPE_VIDEO`.

If that's wrong, stop reading here, but consider some more godocs so that the properties are more clear to the next reader ^^.

If that's correct, though, I'd propose a different tactic:

```
type PBSAdUnit struct {
  ...
  Video PBSVideo
  Banner interface{}
}
```

Benefit being: It eliminates the possibility of a "nonsensical" `PBSAdUnit`. Adapters would then use the `nil`-ness of each property to figure out which media types were valid.